### PR TITLE
MCS-1983 Replaced old webenabled scenes with Eval 7 scenes

### PR DESCRIPTION
**Pull request by ThomasSchellenbergNextCentury**

### CODE REVIEW

File: webenabled/mcs_task_desc.py [hunk 1]
```diff
@@ -35,7 +36,7 @@ class TaskDescription(Enum):
         "balls were added to or subtracted from each side."
     )
 
-    INTERACTIVE_ASYMMETRIC_TOOL_USE = (
+    INTERACTIVE_ASYMMETRIC_TOOL = (
```
Comment (ThomasSchellenbergNextCentury) [inline webenabled/mcs_task_desc.py:39]: I had to rename a few of these to align with the Eval 7 files, but kept the old names as backups.

File: webenabled/mcs_interface.py [hunk 2]
```diff
@@ -372,14 +372,19 @@ def get_task_desc(self, scene_filename):
             f"Attempt to get task description based"
             f"on scene_filename: {scene_filename}")
         scene_type = scene_filename.split('/')[-1].split('0')[0][:-1].upper()
+        # Remove "eval_7_" from the scene name if it's present.
+        scene_type = scene_type.replace('EVAL_7_', '')
+        # Rename passive_agents to passive_agent for simplicity, because I'm
+        # apparently inconsistent with our naming conventions (sorry).
+        scene_type = scene_type.replace('PASSIVE_AGENTS_', 'PASSIVE_AGENT_')
```
Comment (ThomasSchellenbergNextCentury) [inline webenabled/mcs_interface.py:379]: This seemed easier than renaming all of the Eval 7 files.

**Review by NDCACI — APPROVED** [review]

**Comment by phile-caci** [comment]
@NDCACI , I got an error launching this.. are you seeing that too or am I all messed up?

**Comment by phile-caci** [comment]
I commented out the old task names that where changed and reassigned to the new EVAL 7 names, they were causing enumeration errors.

**Review by phile-caci — APPROVED** [review]
it now runs for me after I made the last change in the comment section

**Pull request merged**